co-log-core/README.md

```diff
@@ -1,8 +1,10 @@
 # co-log-core
 
+[![Build status](https://secure.travis-ci.org/kowainik/co-log.svg)](https://travis-ci.org/kowainik/co-log)
 [![Hackage](https://img.shields.io/hackage/v/co-log.svg)](https://hackage.haskell.org/package/co-log-core)
+[![Stackage LTS](http://stackage.org/package/co-log-core/badge/lts)](http://stackage.org/lts/package/co-log-core)
+[![Stackage Nightly](http://stackage.org/package/co-log-core/badge/nightly)](http://stackage.org/nightly/package/co-log-core)
```

Comment:
:thinking: 
The 2 stackage links resolve to a "Not Found" page for me. Has this actually been added to stackage yet?

Reply:
https://www.stackage.org/lts-12.18/hoogle?q=co-log-core gives me nothing as well...